x-pack/plugins/actions/server/builtin_action_types/servicenow/index.test.ts

```diff
@@ -63,12 +62,19 @@ const mockOptions = {
     incidentId: 'ceb5986e079f00100e48fbbf7c1ed06d',
     title: 'Incident title',
     description: 'Incident description',
+    createdAt: '2020-03-13T08:34:53.450Z',
+    createdBy: { fullName: 'Elastic User', username: null },
```

Comment:
nit: check out the user type in `x-pack/plugins/case/common/api/user.ts`, fullName can be undefined and username is always defined a string